consensus/config/src/parameters.rs

```diff
@@ -131,12 +131,16 @@ impl Parameters {
         200
     }
 
-    pub(crate) fn default_sync_last_proposed_block_timeout() -> Duration {
-        Duration::ZERO
+    pub(crate) fn default_sync_last_known_own_block_timeout() -> Duration {
+        if cfg!(msim) {
+            Duration::from_millis(200)
```

Comment:
Simtests are running from `epoch 0` which will make nodes get into amnesia recovery even though it's unnecessary. T to avoid latency penalties we accepted a lower timeout here which should mostly be fine.

Reply:
Or we can disable recover in simtests by default, since it will never happen?

Reply:
I would err on having it run as it doesn't really cost anything significant and it can still exercise the path.

Reply:
Makes sense. Is 200ms enough or does it need to be slightly higher e.g. 500ms?

Reply:
200ms seems to be enough based on current testing, but I am equally fine changing this to 500ms. Let me change that.